test/setup/iam.tf

```diff
@@ -86,21 +86,22 @@ resource "google_folder_iam_member" "int_test_folder" {
   member = "serviceAccount:${google_service_account.int_test.email}"
 }
 
-resource "google_billing_account_iam_member" "int_test_ba" {
-  count = length(local.int_required_ba_roles)
-
-  billing_account_id = var.billing_account
-  role               = local.int_required_ba_roles[count.index]
-  member             = "serviceAccount:${google_service_account.int_test.email}"
-}
```

Comment:
@imrannayer can you revert this resource? I think its complaining that test-sa does now have permission to this BA.